src/NzbDrone.Core/Qualities/Quality.cs

```diff
@@ -67,12 +67,12 @@ public override bool Equals(object obj)
         public static Quality HDTV1080p => new Quality(9,  "HDTV-1080p");
         public static Quality RAWHD => new Quality(10, "Raw-HD");
         //public static Quality HDTV480p    { get { return new Quality(11, "HDTV-480p"); } }
```

Comment:
I don't think we're going too overboard with qualities, but it is more complicated than not adding them. From the initial view we're adding two qualities (remuxes), underneath we're adding quite a bit more, but if you don't care/don't look it stays pretty simple and beyond having a way to completely disable editing the groups I can't think of a way that we could keep it super simple. I don't think we're doing a disservice to normal/new users, but we are definitely making things better for advanced users.

TODO:
- Add `Remux 1080p` and `Remux 2160p`, as their own qualities
- Add `Bluray 480p` (`BDRip` and `BRRip`), Grouped with `DVD`

I'm still not sure about adding `HDTV 480p`, grouped with SDTV because this is a pretty big change since most "SD" content comes from an HDTV feed and would fall under this category, I think this might be the most confusing to normal/new users and I'm not sure they really care they just want their episodes in the smallest size they can.